### PR TITLE
Update 20240828 meetup information

### PR DESCRIPTION
**Pull request by ThreeMonth03**
According to the issue #455 ,this PR adds a new webpage and showcases the schedule for the 8/28 Meetup at NYCU.

### CODE REVIEW

**Comment by yungyuc** [comment]
@kuomc @YenPeiChen07 please help review.

@ThreeMonth03 please update the meetup listing page too.

**Comment by ThreeMonth03** [comment]
> @kuomc @YenPeiChen07 please help review.
> 
> @ThreeMonth03 please update the meetup listing page too.

@yungyuc Thanks for your notification. I fix it.

**Review by YenPeiChen07 — CHANGES_REQUESTED** [review]
@ThreeMonth03 Thank you for adding the meetup information. It looks good overall, but there is a small part need to be updated.

File: content/pages/meetup/2024/0828-nycu.rst [hunk 1]
```diff
@@ -0,0 +1,123 @@
+========================================
+Meetup 2024 August 21st in NYCU
```
[the remark this answers is not in the document]
Comment (ThreeMonth03) [reply content/pages/meetup/2024/0828-nycu.rst]: > The title needs to be updated with the correct date.

Fixed.

[the remark this answers is not in the document]
Comment (ThreeMonth03) [reply content/pages/meetup/2024/0828-nycu.rst]: Thanks for your suggestion. @YenPeiChen07 You could review this pr when you are available.

**Comment by kuomc** [comment]
Hi @ThreeMonth03 , thank you for updating the meetup information and the new meetup page look good, but I found an issue about the GitHub master branch. You use the force push to commit the master branch. I used to merge the commit into the develop branch first to check the new features or pages work well on netlify, and open an new PR to merge commit into master branch from develop branch to publish the new pages. Without special permissions, other people shouldn't be able to directly push or merge the master branch.

Hi @yungyuc @YenPeiChen07 , I didn't see this situation before and I'm not sure if it's good idea to merge the commit into master directly. Please comment if you have some idea.

**Comment by YenPeiChen07** [comment]
> Hi @ThreeMonth03 , thank you for updating the meetup information and the new meetup page look good, but I found an issue about the GitHub master branch. You use the force push to commit the master branch. I used to merge the commit into the develop branch first to check the new features or pages work well on netlify, and open an new PR to merge commit into master branch from develop branch to publish the new pages. Without special permissions, other people shouldn't be able to directly push or merge the master branch.
> 
> Hi @yungyuc @YenPeiChen07 , I didn't see this situation before and I'm not sure if it's good idea to merge the commit into master directly. Please comment if you have some idea.

Thanks, @kuomc, for bringing to our attention.

To protect `master` branch and follow follow [the rule of contribution](https://github.com/sciwork/swportal/blob/e6a8cfd470090eb7582d9d30c1774fa2db905fd7/README.rst#L21), I recommend either create a PR against the `develop` branch or [changing the branch base of this PR](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-base-branch-of-a-pull-request).

From my view, filing another PR against against `develop` branch for tracking and merging would be safer. We can then close this one without directly merging into `master`.

@ThreeMonth03 Could you please help with filing another PR against the `develop` branch?

**Comment by yungyuc** [comment]
@kuomc @YenPeiChen07 with the repository writing permission you may change the destination branch.  I just changed it from `master` to `develop`.

And thanks to @kuomc for highlighting the issue.  @ThreeMonth03 please be careful next time and review the [README](https://github.com/sciwork/swportal/blob/develop/README.rst).

**Comment by ThreeMonth03** [comment]
> @kuomc @YenPeiChen07 with the repository writing permission you may change the destination branch. I just changed it from `master` to `develop`.
> 
> And thanks to @kuomc for highlighting the issue. @ThreeMonth03 please be careful next time and review the [README](https://github.com/sciwork/swportal/blob/develop/README.rst).

My blunder. Thanks for @kuomc explanation, I will notice to make a pull request to the right branch.

**Comment by yungyuc** [comment]
@kuomc @YenPeiChen07 Could you please review again?  We should publish the web page as soon as possible.

**Comment by YenPeiChen07** [comment]
LGTM.

**Pull request merged**